distribution/ddf-common/src/main/resources/etc/default.policy

```diff
@@ -0,0 +1,157 @@
+priority "grant";
+// todo symlinks
+
+deny {
+    // Read, write, or execute any file
+    permission java.io.FilePermission "<<ALL FILES>>", "read, write, execute";
+
+    // Deny deletion of the security policy file through any Java code
+    permission java.io.FilePermission "${ddf.home}${/}etc${/}default.policy", "delete";
+
+    permission java.util.PropertyPermission "javax.net.ssl.*", "read, write";
+
+    // Change current security manager
+    permission java.lang.RuntimePermission "setSecurityManager";
+
+    // Modify application permissions at will.
+    permission java.security.SecurityPermission "setPolicy, insertProvider, removeProvider.*, clearProviderProperties.*, putProviderProperty.*, removeProviderProperty.*";
+
+    // Load classes into any protection domain
+    permission java.lang.RuntimePermission "createClassLoader";
+};
+
+//
+// DDF Libraries have broad file permissions within the install directory
+//
+grant
+  codeBase "file:${ddf.home.policy}system/ddf/-" {
+    permission java.io.FilePermission "${ddf.home}", "read, write";
+    permission java.io.FilePermission "${ddf.home}${/}etc${/}-", "read, write";
```

Comment:
Felix config installer (and Steve's replacement) will also need delete for .cfg and .config files located under etc (not recursive)